### ISSUE TITLE
Riemann health stopped updating disk usage

### ISSUE THREAD

**Issue by chris-martin** [riemann/riemann-tools#148]
I just noticed that one of my machine's disk usage was being reported by riemann-health at 50%, even though it was actually at 69%. It was still sending events regularly, but the metric was wrong.

I restarted the riemann-health daemon, and the metric updated to the correct number.

**Comment by jamtur01** [comment]
@chris-martin What version of riemann-health and what platform?

**Comment by chris-martin** [comment]
How do I get the version? I don't see anything resembling a `--version` flag on `riemann-health`.

It's running on Ubuntu 14.04 on AWS.

**Comment by jamtur01** [comment]
```shell
$ gem list riemann-tools
```

**Comment by chris-martin** [comment]
0.2.8

**Comment by jamtur01** [comment]
Can you paste the result of:

```shell
$ df -P --exclude-type=iso9660
```

Can you tell me which filesystem in the results had the issue and if you have a copy of the Riemann event that'd be great too.

**Comment by chris-martin** [comment]
```
Filesystem     1024-blocks     Used Available Capacity Mounted on
/dev/xvda1        51466360 33696208  15573660      69% /
none                     4        0         4       0% /sys/fs/cgroup
udev               1019260        8   1019252       1% /dev
tmpfs               204848      372    204476       1% /run
none                  5120        0      5120       0% /run/lock
none               1024228       96   1024132       1% /run/shm
none                102400        0    102400       0% /run/user
```

`disk /` is the only one that's being reported on

**Comment by chris-martin** [comment]
I don't have the old events. If the event data would be useful, I just need to know how to capture it.

**Comment by jamtur01** [comment]
Are you running riemann-health in an init script? cron job?

**Comment by chris-martin** [comment]
Upstart init script:

```
start on runlevel [2345]
stop on runlevel [!2345]
kill timeout 5
respawn

script
exec riemann-health -l 30 -h [...]
end script
```

**Comment by jamtur01** [comment]
Should be fixed by #153

**Comment by mhluongo** [comment]
Thanks @jamtur01!
